packages/data-context/src/actions/CodegenActions.ts

```diff
@@ -33,6 +33,7 @@ export class CodegenActions {
       let result = parseReactComponent(src, {
         resolver: findAllWithLink(exportResolver, reactDocgenResolvers),
         babelOptions: {
+          configFile: false,
```

Comment:
This option disables the use of a config file for Babel. this is what we want since we're calling it directly, we don't want to pick up any config file that might be in the project

https://babeljs.io/docs/options#configfile

Reply:
is it possible to add a test for this?

Reply:
@AtofStryker I added a test in `CodegenActions.spec.ts` just to make sure that we pass this option to Babel. I went down the road of trying to do a system test for this instead, but couldn't get it to work

Reply:
its probably hard to do a system test since its code generation for a component? Have you tried a cy-in-cy test? That would give us the most coverage here

Reply:
@astone123 any luck with the cy-in-cy test?

Reply:
no I haven't looked at it yet, will try it out this week